rsdoctor/next-minimal/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "next-app",
```

Comment:
there are many examples in this repo, so we'd better set a unified name for all Rsdoctor examples, such as `rsdoctor-next-app`.

Reply:
done~